pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[project]
+dynamic = ["version", "dependencies", "readme"]
+name = "cornice"
+description = "Define Web Services in Pyramid."
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+    "Framework :: Pylons",
+    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+]
+keywords = ["web pyramid pylons"]
+authors = [
+  {name = "Mozilla Services", email = "services-dev@mozilla.org"},
+]
+
+[project.urls]
+Repository = "https://github.com/Cornices/cornice"
+
+[tool.setuptools_scm]
+# can be empty if no extra settings are needed, presence enables setuptools_scm
+
+[tool.setuptools.dynamic]
+dependencies = { file = ["requirements.in"] }
+readme = {file = ["README.rst", "CONTRIBUTORS.txt"]}
+
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project.optional-dependencies]
+dev = [
+    "ruff",
+    "pytest",
+    "pytest-cache",
+    "pytest-cov",
+    "webtest",
+    "marshmallow<4",
+    "colander",
+]
```

Comment:
I recommend pinning version ranges to avoid breaking if one of these deps introduces a breaking change — let’s assume they use [semver](https://semver.org/).